lib/private/AppFramework/Utility/SimpleContainer.php

```diff
@@ -105,23 +106,18 @@ public function resolve($name) {
 		}
 	}
 
-
-	/**
-	 * @param string $name name of the service to query for
-	 * @return mixed registered service for the given $name
-	 * @throws QueryException if the query could not be resolved
-	 */
-	public function query($name) {
+	public function query(string $name, bool $autoload = true) {
 		$name = $this->sanitizeName($name);
 		if ($this->offsetExists($name)) {
 			return $this->offsetGet($name);
-		} else {
+		} else if ($autoload === true) {
```

Comment:
```suggestion
		} else if ($autoload) {
```

it can only be bool :wink: